packages/arb-token-bridge-ui/src/types/ChainQueryParam.ts

```diff
@@ -16,15 +17,15 @@ const chainQueryParams = [
   'arbitrum-nova',
   'arbitrum-goerli',
   'arbitrum-sepolia',
-  'xai',
   'stylus-testnet',
-  'xai-testnet',
+  'local',
+  'arbitrum-local',
   'custom-localhost',
   'arbitrum-localhost'
 ] as const
 
 export type ChainKeyQueryParam = (typeof chainQueryParams)[number]
-export type ChainQueryParam = ChainKeyQueryParam | ChainId | number
```

Comment:
Unfortunately `| string` kinda ruins the type here, but otherwise we will have to add another slug to this list each time we add a new Orbit chain.

I personally prefer less diff for adding new chains than having a bit stronger type here. Happy to hear other points